src/viam/sdk/services/motion/server.cpp

```diff
@@ -52,10 +49,8 @@ ::grpc::Status MotionServer::MoveOnMap(
         const auto& destination = pose::from_proto(request->destination());
         const auto& component_name = Name::from_proto(request->component_name());
         const auto& slam_name = Name::from_proto(request->slam_service_name());
-        const bool success = motion->move_on_map(std::move(destination),
-                                                 std::move(component_name),
-                                                 std::move(slam_name),
-                                                 helper.getExtra());
+        const bool success =
+            motion->move_on_map(destination, component_name, slam_name, helper.getExtra());
```

Comment:
Ditto

Reply:
Also, why are those variables declared as `&`s up on 49-51?

Reply:
Good question on `&`s! Not sure, deleted.

I've un-`const`ed the variables and added back the moves (except for `destination`; `pose` is trivially copyable so I maintained the `const`ness of it).